.github/workflows/checks.yml

```
@@ -28,3 +28,33 @@ jobs:
       - run: yarn
       - run: yarn run prettify:diff
       - run: yarn run lint
+  mobile-tests:
+    name: Mobile Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '12'
+          check-latest: true
+      - name: Install Yarn dependencies
+        run: yarn
+      - name: Build Yarn packages
+        run: yarn build --ignore docs
+      - name: Verify locales if using l10n/* branch
+        run: |
+          if [[ "${GITHUB_REF##*/}" != l10n/* ]]; then
```

Looks like it doesn't do the expected when the branch has more `/`

```shell
bash-3.2$ export GITHUB_REF="refs/heads/feature/something"
bash-3.2$ echo "${GITHUB_REF##*/}"
something
```
but we should have `feature/something` instead.

I propose the following instead, and using `GITHUB_HEAD_REF` because we want this check done only from a PR originating from an `l10n/*` branch
```suggestion
          if [[ "${GITHUB_HEAD_REF#refs/heads/}" != l10n/* ]]; then
```

---

fixed! thanks!